doc/source/cluster/faq.rst

```diff
@@ -0,0 +1,26 @@
+===
+FAQ
+===
+
+These are some Frequently Asked Questions that we've seen pop up for using Ray clusters.
+If you still have questions after reading this FAQ,  please reach out on
+`our Discourse <https://discuss.ray.io/>`__!
+
+Do Ray clusters support multi-tenancy?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, you can run multiple :ref:`jobs <jobs-overview>` simultaneously in a Ray cluster
+but it's NOT recommended in production.
+Instead, you can reuse the same cluster for running multiple jobs one at a time.
+The reason is that Ray currently still misses some features for multi-tenancy in production:
```

Comment:
How about let's have two questions.

1. Do ray clusters support multi-tenancy?
2. I have multiple Ray users. What's the right way to deploy Ray for them?

Reply:
Updated. PTAL.